editor/src/components/common/shared-strategies/convert-to-flex-strategy.ts

```diff
@@ -29,17 +37,15 @@ type FlexAlignItems = 'center' | 'flex-end'
 export function convertLayoutToFlexCommands(
   metadata: ElementInstanceMetadataMap,
   elementPaths: Array<ElementPath>,
+  allElementProps: AllElementProps,
 ): Array<CanvasCommand> {
   return elementPaths.flatMap((path) => {
     const parentInstance = MetadataUtils.findElementByElementPath(metadata, path)
     if (parentInstance == null) {
       return []
     }
 
-    const childrenPaths = MetadataUtils.getChildrenPathsUnordered(metadata, path)
-
-    if (areAnyChildrenNonDomElement(metadata, childrenPaths)) {
-      // This is a known limitation and future TODO. we must early return now to avoid bizarro layouts
+    if (MetadataUtils.isConditionalFromMetadata(parentInstance)) {
```

Comment:
could you put a comment here describing what problems we need solving so we don't have to show the toast here?

Reply:
added the comment in https://github.com/concrete-utopia/utopia/pull/3557/commits/89bc4eec43afaa8c639b80103a7a8b84a4293fbb

Reply:
commented in https://github.com/concrete-utopia/utopia/pull/3557/commits/89bc4eec43afaa8c639b80103a7a8b84a4293fbb